week5-api/src/App.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+
+class CharacterInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      character: {}
+    };
+  }
+
+  componentDidMount() {
+    const characterId = this.props.match.params.characterId;
+    fetch('https://rickandmortyapi.com/api/character/'+characterId)
```

Comment:
cleaner way is fetch(``https://rickandmortyapi.com/api/character/${characterId}``) 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals